include/lbann/base.hpp

```diff
@@ -39,24 +39,34 @@
 // Typedefs for Elemental matrices
 using EGrid      = El::Grid;
 using Grid       = El::Grid;
-using Mat        = El::Matrix<lbann::DataType>;
+using AbsMat     = El::AbstractMatrix<lbann::DataType>;
+using Mat        = El::Matrix<lbann::DataType, El::Device::CPU>; // Temporarily define as CPUMat
```

Comment:
I think we should indicate that this typedef is deprecated by moving it next to `DistMat`.

Reply:
Done